Vagrantfile

```diff
@@ -0,0 +1,62 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+ENV['VAGRANT_DEFAULT_PROVIDER'] ||= "virtualbox"
+DOCKER_PORT=2376
+UNICORN_PORT=3000
+VAGRANTFILE_API_VERSION = "2"
+
+$docker_setup=<<CONF
+cat > /etc/default/docker << 'EOF'
+DOCKER_OPTS="-H 0.0.0.0:#{DOCKER_PORT} -H unix:///var/run/docker.sock"
+EOF
+service docker restart
+CONF
+
+unless Vagrant.has_plugin?("vagrant-cachier")
+  puts "WARNING: vagrant-cachier plugin is not installed! It really speeds this up..."
+  puts "         Install using 'vagrant plugin install vagrant-cachier'"
+end
+
+unless Vagrant.has_plugin?("vagrant-persistent-storage")
+  puts "WARNING: vagrant-persistent-storage not installed! It really speeds up docker image mgmt"
```

Comment:
We may not, but `mgmt` made it look like you were adhering to one (or something similar)… But also, 80 character lines are nice, m'kay? (I'm not actually saying we should adhere religiously to one, but if a nice can be nicely made to fit, why not?)